src/Routes/Devices/DeviceTable.js

```diff
@@ -213,6 +213,7 @@ const DeviceTable = ({
   fetchDevices,
   isSystemsView = false,
   isAddSystemsView = false,
+  ariaLabel,
```

Comment:
According to https://reactjs.org/docs/accessibility.html#wai-aria, `ariaLabel` should be `aria-label`.